spec/features/thredded/visitor_signs_in_spec.rb

```diff
@@ -1,14 +1,19 @@
 # frozen_string_literal: true
 require 'spec_helper'
 
-feature 'Signing in' do
-  scenario 'Visitor with existing account signs in' do
-    create(:user, name: 'joe', email: 'joe@example.com')
-    visit new_user_session_path
-    fill_in 'name', with: 'joe'
-    click_button 'Sign in'
+feature 'Signing in with existing account' do
+  let(:user) { PageObject::User.new(create(:user, name: 'joe', email: 'joe@example.com')) }
 
-    expect(page).to have_content('joe')
-    expect(page).to have_content('Sign out')
+  scenario 'directly' do
+    user.log_in
+    expect(user).to be_signed_in
+  end
```

Comment:
nit: Having only "directly" as the scenario's title seems a little on the vague side. What do you think about something a little more explicit - `'user signs in'`